routes/index.tsx

```diff
@@ -88,6 +89,24 @@ export default function HomePage(props: PageProps<HomePageData>) {
       <Head href={props.url.href} />
       <main class="flex-1 p-4">
         <TimeSelector url={props.url} />
+        {props.data.items.length === 0 && (
+          <>
+            <div class="flex flex-col justify-center items-center gap-2">
+              <div class="flex justify-center gap-2 pt-16">
+                <Info class="w-6 h-6" />
```

Comment:
Perhaps, the icon would look nicer placed on the line above and centered. WDYT? I don't feel strongly.

Reply:
Yeah! agreed.